ci/build.py

```diff
@@ -23,7 +23,11 @@
 """
 
 __author__ = 'Marco de Abreu, Kellen Sunderland, Anton Chernov, Pedro Larroy'
-__version__ = '0.1'
+__version__ = '0.8'
+
+def pip_install(pkg):
```

Comment:
We shouldn't install anything during a build.py run, all requirements should be there already

Reply:
agree, just trying to see if I could get Jenkins to run. Will remove it before final merge.

Reply:
⚡